### PR TITLE
Avoid SL4J noise by triggering logger init from single thread

### PR DESCRIPTION
**Pull request by johanandren**
References #28673

### CODE REVIEW

**Review by raboof — APPROVED** [review]
I'm not sure I understand exactly what's going on here (http://www.slf4j.org/codes.html#replay is a bit vague on why it's a problem that you'd start logging "when already multithreaded"), but :+1: if this fixes that noise :smile:

File: akka-actor-testkit-typed/src/main/scala/akka/actor/testkit/typed/scaladsl/ActorTestKit.scala [hunk 1]
```diff
@@ -135,6 +136,9 @@ final class ActorTestKit private[akka] (val name: String, val config: Config, se
   implicit def testKitSettings: TestKitSettings =
     settings.getOrElse(TestKitSettings(system))
 
+  // avoid slf4j noise by touching it first from single thread #28673
+  LoggerFactory.getLogger(getClass).debug("Starting ActorTestKit")
```
Comment (patriknw) [inline akka-actor-testkit-typed/src/main/scala/akka/actor/testkit/typed/scaladsl/ActorTestKit.scala]: include `name` for additional value

**Review by patriknw — APPROVED** [review]
LGTM, with small addition

File: akka-actor-testkit-typed/src/main/scala/akka/actor/testkit/typed/scaladsl/ActorTestKit.scala (hunk 1 above)
Comment (johanandren) [reply akka-actor-testkit-typed/src/main/scala/akka/actor/testkit/typed/scaladsl/ActorTestKit.scala]: Had bad luck thinking when I passed the class... fixed.

**Pull request merged**